umm-spec-lib/src/cmr/umm_spec/migration/version_migration.clj

```diff
@@ -17,13 +19,30 @@
 ;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
 ;;; Utility Functions
 
+(defn- customized-compare
+  "Compare begin and end.
```

Comment:
I would just comment why we need to do the customized compare, i.e. "1.1" vs "1.10"